examples/aoc2024/day17/part2.jou

```diff
@@ -28,14 +28,14 @@ def get_first_output(code: int*, reg_a: long) -> int:
         elif opcode == 1:  # bxl = B bitwise Xor with Literal
             regs[1] ^= operand
         elif opcode == 2:  # bst = B Set value and Truncate to 3 bits
-            regs[1] = do_combo_op(operand, regs) % 8
+            regs[1] = do_combo_op(operand, regs) & 0b111
         elif opcode == 3:  # jnz = Jump if NonZero
             if regs[0] != 0:
                 ip = &code[operand]
         elif opcode == 4:  # bxc = B Xor C
             regs[1] ^= regs[2]
         elif opcode == 5:  # out = OUTput value
-            return (do_combo_op(operand, regs) % 8) as int
+            return (do_combo_op(operand, regs) & 0b111) as int
```

Comment:
I think `as int` is unnecessary due to Jou's rules for determining the type of `&`. It is the smaller of the two types (or the unsigned one if the same size).

Reply:
```suggestion
            return do_combo_op(operand, regs) & 0b111
```